src/language-js/print/array.js

```diff
@@ -75,24 +88,42 @@ function printArray(path, options, print) {
         return element[itemsKey] && element[itemsKey].length > 1;
       });
 
+    const shouldUseConciseFormatting =
+      n.elements.length > 1 &&
+      n.elements.every(
+        (element) =>
+          element &&
+          (isNumericLiteral(element) ||
+            (isSignedNumericLiteral(element) &&
+              !hasComment(element.argument))) &&
```

Comment:
Is this line tested?

**Prettier pr-10106**
[Playground link](https://deploy-preview-10106--prettier.netlify.app/playground/#N4Igxg9gdgLgprEAucACAvKg2gHSq1AWlQHoAqMk1ANmoBob7UBWAdgYBZnP3UBmPg2aDUADgBMDUQEYGrJrSG8uPBgKEiJU2anHiB+vAF0A3CDogIABxgBLaAGdkoAIYAnNxADuABXcInFBcAGy8XAE8nCwAjNxcwAGs4GABlK3jbKABzZBg3AFc4CzgAW2i4ABMKyoAZF2z8lyy4ADEINxKXGDts5BAXfJgIcxAACxgS4IB1Udt4B3SwOBSAudsANznwvrAHKJBMhzg3GB84rM7kADMQo4sAKwcADwAhOMTklJcSuBrMuGutyKIEeTxSmSywTgAEV8hB4IDgncQOk3Ec3H1oi5ysERlY3JkYFNbBUYKNkKIAAwWfEQI5TOJWPr4uDo9YAiwARzh8DO1kC-QchCgcEqlRGbjg3NskrOTUuSBuSOBRxKtlyBRVEKhsPhAMVQIsMGxxNJ5KQkhAeRctmCEIAwhASgqQKzmCN8kcACrYwJK5HrQoASSg1VgKTABJsAEFQykYOEoYijgBfFNAA)
```sh
--parser babel
```

**Input:**
```jsx
c = [
  - /**/ 66, 66, 57, 45, 47, 33, 53, 82, 81, 76, 66, 57, 45, 47, 33, 53, 82, 81, 223323
];
```

**Output:**
```jsx
c = [
  -(/**/ 66),
  66,
  57,
  45,
  47,
  33,
  53,
  82,
  81,
  76,
  66,
  57,
  45,
  47,
  33,
  53,
  82,
  81,
  223323,
];

```

Reply:
I'm fine with this, just make sure it's tested.

Reply:
There was a similar test for a line comment, but I've added this one too.